x-pack/test/security_solution_cypress/cypress/tasks/lists.ts

```diff
@@ -160,3 +173,95 @@ export const importValueList = (
 ) => {
   return cy.fixture<string>(file).then((data) => uploadListItemData(file, type, data));
 };
+
+export const openValueListItemsModal = (listId: string) => {
+  return cy.get(`[data-test-subj="show-value-list-modal-${listId}"]`).click();
+};
+
+export const searchValueListItemsModal = (search: string) => {
```

Comment:
Not related with your changes because I saw that this was done like that before, but, for future refactors it would be great if we differentiate the tasks we do using the API over the ones we do through the UI. All the tasks that are done through the API are implemented in the `x-pack/test/security_solution_cypress/cypress/tasks/api_calls` folder.